test/form_builder/general_test.rb

```diff
@@ -136,6 +136,24 @@ def with_custom_form_for(object, *args, &block)
     assert_select 'form input#user_description.string'
   end
 
+  test 'builder generates text areas for hstore columns' do
+    with_form_for @user, :hstore
+    assert_no_select 'form input#user_hstore.string'
+    assert_select 'form textarea#user_hstore.text'
+  end
+
+  test 'builder generates text areas for json columns' do
+    with_form_for @user, :json
+    assert_no_select 'form input#user_json.string'
+    assert_select 'form textarea#user_json.text'
```

Comment:
The reason that the tests are failing is because in other rails version (I used Rails 4.0 of example) the form is generating this path: `form textarea#user_hstore.json` - I don't no why, I didn't investigate it 🙈.

Maybe we can use just  `form textarea#user_hstore` for these tests? 🤔 Wdty @rafaelfranca ?

Reply:
It is only 4.0? At this point I'm more inclined to stop supporting anything that is not Rails 5.0+

Reply:
I used Rails 5.1 in my tests. I will check if it's ok in the 5.0 version. If so, can we remove the older versions?.

Reply:
:+1: